prowler/providers/kubernetes/lib/audit_info/models.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from typing import Any, Optional
+
+from kubernetes import client
+
+
+@dataclass
+class Kubernetes_Audit_Info:
```

Comment:
```suggestion
class KubernetesAuditInfo:
```

Can we do that to comply with Python naming standards?

Reply:
We will have to change all the naming in the other providers then...